terraform/shared/main.tf

```diff
@@ -106,3 +106,7 @@ module "securityhub" {
 module "ecr" {
   source = "./modules/ecr"
 }
+
+module "s3" {
```

Comment:
could we change the name of this to be like s3_policy or something, so that we don't think its just a bucket